packages/runtime/client/rest/src/index.ts

```diff
@@ -77,20 +78,28 @@ export interface RestAPI {
    * Gets a single contract by id
    * @param contractId The id of the contract to get
    * @throws DecodingError - If the response from the server can't be decoded
-   * @see {@link https://docs.marlowe.iohk.io/api/get-contracts-by-id}
+   * @see {@link https://docs.marlowe.iohk.io/api/get-contract-by-id | The backend documentation}
    */
-  // TODO: Jamie, remove the `s from the end of the endpoint name in the docs site
-  getContractById(contractId: ContractId): Promise<ContractDetails>; // - https://docs.marlowe.iohk.io/api/get-contracts-by-id
-  //   In the docs site, after getContractsById there is https://docs.marlowe.iohk.io/api/create-contracts-by-id, not sure what method it corresponds or what is that method supposed to do.
+  getContractById(contractId: ContractId): Promise<ContractDetails>;
   /**
    * Submits a signed contract creation transaction
+   * @see {@link https://docs.marlowe.iohk.io/api/submit-contract-to-chain | The backend documentation}
    */
   submitContract(
     contractId: ContractId,
     txEnvelope: TextEnvelope
   ): Promise<void>;
-  //
-  //   getTransactionsForContract: Transactions.GET; // - https://docs.marlowe.iohk.io/api/get-transactions // TODO: Jamie, lets unify names
+  /**
+   * Gets a paginated list of  {@link contract.TxHeader } for a given contract.
+   * @see {@link https://docs.marlowe.iohk.io/api/get-transactions-for-contract | The backend documentation }
+   */
+  // DISCUSSION: What should this return when contractId is not found? Currently it throws an exception
+  //             with an AxiosError 404, we could return a nullable value or wrap the error into a custom
+  //             ContractNotFound error and specify it in the docs.
```

Comment:
cc @jhbertra @nhenin

Reply:
It's fine if we throw an exception here - the caller should be sure the contract exists, not use this as a means of testing for its existence. Wrapping in a custom error is not a bad idea.